### PR TITLE
Doc lint fixes.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: docs/appliance.rst [hunk 1]
```diff
@@ -257,6 +257,7 @@ environments such as Amazon Web Services and Google Compute Engine.
 
 .. _Galaxy: http://galaxyproject.org/
 .. _Docker: https://www.docker.com/
+.. _Conda: http://conda.pydata.org/
```
Comment (nsoranzo) [inline docs/appliance.rst]: It's https://conda.io/ nowadays.

File: planemo/network_util.py [hunk 2]
```diff
@@ -33,10 +33,11 @@ def wait_http_service(url, timeout=None):
...
 def wait_net_service(server, port, timeout=None):
-    """ Wait for network service to appear
-        @param timeout: in seconds, if None or 0 wait forever
-        @return: True of False, if timeout is None may return only True or
-                 throw unhandled network exception
+    """ Wait for network service to appear.
+
+    @param timeout: in seconds, if None or 0 wait forever
+    @return: True of False, if ``timeout`` is ``None`` may return only True or
+    throw unhandled network exception
```
Comment (nsoranzo) [inline planemo/network_util.py]: Shouldn't that be indented?

Comment (jmchilton) [reply planemo/network_util.py]: I tried the old way, I tried two spaces, 4 spaces, Sphinx hates everything but this.

Comment (peterjc) [reply planemo/network_util.py]: Typo in the line above?

``True of False`` --> ``True or False``

Comment (nsoranzo) [reply planemo/network_util.py]: Shouldn't these start with a `:` http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists
```
:param timeout: ...
:return: ...
```

Comment (jmchilton) [reply planemo/network_util.py]: Revised the docs to implement this I think.

**Pull request merged**